src/web-experiments-types.ts

```diff
@@ -0,0 +1,38 @@
+export interface WebExperimentTransform {
+    attributes:
+        | {
+              name: string
+              value: string
+          }[]
+        | null
```

Comment:
making this value nullable instead of optional is interesting; what's the significance of going with this approach over something like this?
```
attributes?: { name: string; value: string }[]
```
Is making this value nullable vs undefined important?  Feels like we could avoid the extra null checks elsewhere by making this optional.

Reply:
TBH, I don't have a strong preference here. I like your approach better, will adopt.